packages/core/README.md

```diff
@@ -14,7 +14,7 @@ npm install --save @enact/core
 
 ## Test
 
-Tests are implemented in mocha and are run with the Karma runner.
+Unit tests are implemented with Jest. To execute them:
```

Comment:
How about
```suggestion
Unit tests are implemented in Testing Library and are run with Jest. To execute them:
```

Reply:
I just made the same as what was in the Sandstone, but I prefer your suggestion. I will also update the same section in the Sandstone.